unstructured/documents/html.py

```diff
@@ -97,6 +97,7 @@ def _read(self) -> List[Page]:
             return self._pages
         logger.info("Reading document ...")
         pages: List[Page] = []
+        etree.strip_elements(self.document_tree, ["script"])
```

Comment:
Good call 👍 This is a much conciser solution.

Reply:
LGTM!